scene/resources/texture.h

```diff
@@ -474,15 +492,21 @@ class Texture3D : public Texture {
 protected:
 	static void _bind_methods();
 
-	TypedArray<Image> _get_data() const;
-
-public:
-	virtual Image::Format get_format() const = 0;
-	virtual int get_width() const = 0;
-	virtual int get_height() const = 0;
-	virtual int get_depth() const = 0;
-	virtual bool has_mipmaps() const = 0;
-	virtual Vector<Ref<Image>> get_data() const = 0;
+	TypedArray<Image> _get_datai() const;
```

Comment:
What does `i` stand for here?

Reply:
internal I suppose, its not user exposed and _get_data is virtual already.